skimage/filters/thresholding.py

```diff
@@ -766,27 +766,36 @@ def _mean_std(image, w):
            Retrieval XV, (San Jose, USA), Jan. 2008.
            DOI:10.1117/12.767755
     """
-    if w == 1 or w % 2 == 0:
```

Comment:
This check needs to come back, but you need to check _every_ element of the tuple for this.